pkg/queue/manager_test.go

```diff
@@ -195,6 +196,44 @@ func TestUpdateClusterQueue(t *testing.T) {
 	}
 }
 
+func TestCohortCycle(t *testing.T) {
```

Comment:
```suggestion
func TestMoveWorkloadsCohort(t *testing.T) {
```
nit, or `TestQueueInadmissibleWorkloadsForCohort`

Reply:
Renamed to TestMoveWorkloadsCohortCycle, since the important part of this test is that this function doesn't misbehave when there's a cycle.